analysis/deep2_r23_o32/general.py

```diff
@@ -29,7 +29,8 @@ def run_grid_r23_o32_analysis(dataset, n_split=3, y_correction=False,
            Version for spectrum in zoom_and_gauss_general. Default: False
     :param adaptive: bool. Set to use adaptive binning, otherwise equal
            log(R23) bins. Default: True
-    :param apply_dust: bool. Apply dust correction. Default: False
+    :param apply_dust: bool. Apply dust correction. Default: True
+                    By setting apply_dust=true, all cases are run
```

Comment:
Wouldn't it make more sense to still keep the default as `False` and then do: `general.run_grid_r23_o32_analysis(dataset, apply_dust=True)`